tools/modify_fsurdat/fsurdat_modifier

```diff
@@ -28,10 +28,9 @@ This wrapper script accesses
 
 Instructions
 ------------
-To run on Cheyenne/Casper
-1) Load the following to your local environment:
-module load python
-ncar_pylib
+To run on cheyenne/casper/izumi
+1) Activate the following environment:
```

Comment:
> This would be similar to what you have for the mesh modifier below.

From looking at this branch versus the reduce_redundancy branch, I see that the README for modify_fsurdat and your recommended changes to it are coming in with the #1714 PR.